ledger/apply/keyreg.go

```diff
@@ -76,7 +81,7 @@ func Keyreg(keyreg transactions.KeyregTxnFields, header transactions.Header, bal
 		record.VoteFirstValid = keyreg.VoteFirst
 		record.VoteLastValid = keyreg.VoteLast
 		record.VoteKeyDilution = keyreg.VoteKeyDilution
-		if balances.ConsensusParams().EnableStateProofKeyregCheck {
+		if params.EnableStateProofKeyregCheck {
```

Comment:
similarly, we should get rid of this and consolidate on the place where VoteID and SelectionID are set

Reply:
same as previous comment

Reply:
fixed in e43e25156b233c9c8c5c764282f3e8e42022d780